src/storage/patch/RepresentationPatchHandler.ts

```diff
@@ -41,6 +44,12 @@ export class RepresentationPatchHandler extends PatchHandler {
     // Patch it
     const patched = await this.patcher.handleSafe({ patch, identifier, representation });
```

Comment:
Should the canHandle be then the following:
```typescript
    await this.patcher.canHandle({ identifier, patch, representation: undefined });
```

Reply:
Nevermind, just realized that the representation gets created in the `RepresentationPatchHandler` and it doesn't receive that as input. There might be some solutions but they wouldn't be very useful so this can be left as is.